core/domain/exp_domain.py

```diff
@@ -2890,6 +3961,19 @@ def from_yaml(cls, exploration_id, yaml_content):
     def from_untitled_yaml(cls, exploration_id, title, category, yaml_content):
         """Creates and returns exploration from a YAML text string. This is
         for importing explorations using YAML schema version 9 or earlier.
+
+        Args:
+            exploration_id: str. The id of the exploration.
+            title: str. The exploration title.
+            category: str. The exploration category.
+            yaml_content: str. The YAML representation of the exploration.
+
+        Returns:
+            Exploration. The corresponding exploration domain object.
+
+        Raises:
+            Exception: The initial schema version of exploration is less than
+            or equal to 9.
```

Comment:
Indent by 4, since this is a continuation line and there can be multiple exceptions.